### PR TITLE
doc: Compare CloudQuery vs CSPMs

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: docs/cq_vs_others/cspms.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+
+CSPMs are set of closed-source enterprise products to monitor, detect and responsd to threats and mis-configurations in the cloud.
+
+**Key Differences:**
+- **Open Source Pluggable Architecture**: CloudQuery has an open-source [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to either official APIs or even create your own integrations to proprietry APIs or SaaS applications.
+- **Vanilla PostgreSQL**: CloudQuery uses vanilla PostgreSQL which makes it play nicely with the whole SQL eco-system and gives you the ability to re-use other tools like Grafana/BI. CSPMs are usually using a proprietary query languages and databases and thus doesn't give you the ability to re-use other tools easily.
+- **Policy Language**: CQ uses an HCL configuration with SQL as the query engine to define rules. CSPMs use custom query languages and closed source databases.
+- **Pricing**: CloudQuery is open-source and thus you will pay only for the hosting of your PostgreSQL (you can use RDS, or any other managed version) and the compute for running [CQ binary](../deployment/helm-chart). CSPMs can sometimes cost alot of money for big accounts.
```
Comment (bbernays) [inline docs/cq_vs_others/cspms.md:9]:
```suggestion
- **Pricing**: CloudQuery is open-source and thus you will pay only for the hosting of your PostgreSQL (you can use RDS, or any other managed version on whatever cloud provider you would like) and the compute for running [CQ binary](../deployment/helm-chart). Nearly all CSPMs require long term contracts and have variable pricing which makes ver.
```

File: docs/cq_vs_others/cspms.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# CloudQuery vs CSPMs (Cloud Security Posture Management)
+
+CSPMs are set of closed-source enterprise products to monitor, detect and responsd to threats and mis-configurations in the cloud.
+
+**Key Differences:**
+- **Open Source Pluggable Architecture**: CloudQuery has an open-source [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to either official APIs or even create your own integrations to proprietry APIs or SaaS applications.
```
Comment (bbernays) [inline docs/cq_vs_others/cspms.md:6]:
```suggestion
- **Open Source Pluggable Architecture**: CloudQuery has an open-source [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to existing providers or you can create your own providers in order to grab data from proprietary APIs or other SaaS applications.
```

File: docs/cq_vs_others/cspms.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- **Vanilla PostgreSQL**: CloudQuery uses vanilla PostgreSQL which makes it play nicely with the whole SQL eco-system and gives you the ability to re-use other tools like Grafana/BI. CSPMs are usually using a proprietary query languages and databases and thus doesn't give you the ability to re-use other tools easily.
```
Comment (bbernays) [inline docs/cq_vs_others/cspms.md:7]: What does BI standard for?

Comment (bbernays) [inline docs/cq_vs_others/cspms.md:7]: Do any CSPMs not use a proprietary query language? 
Do any CSPMs give users direct access to the DB? As far as I am aware, none give access to the raw data

Comment (yevgenypats) [reply docs/cq_vs_others/cspms.md:7]: Yeah - none.

Comment (yevgenypats) [reply docs/cq_vs_others/cspms.md:7]: Business intelligence ?

File: docs/cq_vs_others/cspms.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# CloudQuery vs CSPMs (Cloud Security Posture Management)
+
+CSPMs are set of closed-source enterprise products to monitor, detect and response to threats and mis-configurations in the cloud.
+
+**Key Differences:**
+- **Open Source Pluggable Architecture**: CloudQuery has an open-source [pluggable architecture](https://hub.cloudquery.io) which means you can contribute missing resources to either official APIs or even create your own integrations to proprietry APIs or SaaS applications.
+- **Vanilla PostgreSQL**: CloudQuery uses vanilla PostgreSQL which makes it play nicely with the whole SQL eco-system and gives you the ability to re-use other tools like Grafana/BI. CSPMs are usually using a proprietary query languages and databases and thus doesn't give you the ability to re-use other tools easily.
+- **Policy Language**: CQ uses an HCL configuration with SQL as the query engine to define rules. CSPMs use custom query languages and closed source databases.
```
Comment (bbernays) [inline docs/cq_vs_others/cspms.md:8]: I wouldn't mention HCL... That is a trivial thing compared to the SQL part

File: docs/cq_vs_others/cspms.md (hunk 3 above)
Comment (bbernays) [reply docs/cq_vs_others/cspms.md:7]: In that case, maybe we can say `All broadly adopted business intelligence tools support grabbing data from a postgres db, which means you won't have to change your existing tooling in order to visualize your data`

File: docs/cq_vs_others/cspms.md (hunk 4 above)
Comment (yevgenypats) [reply docs/cq_vs_others/cspms.md:8]: agree

File: docs/cq_vs_others/cspms.md (hunk 3 above)
Comment (yevgenypats) [reply docs/cq_vs_others/cspms.md]: updated

**Pull request merged**